nginx.conf

```diff
@@ -53,7 +53,7 @@ http {
         }
 
         location /latest-launcher-template {
-            return 301 https://raw.githubusercontent.com/openshiftio/launchpad-templates/v8/openshift/launchpad-template.yaml;
+            return 301 https://raw.githubusercontent.com/openshiftio/launchpad-templates/v9/openshift/launchpad-template.yaml;
```

Comment:
Am I correct to assume this line will always have to be updated with the new release? If so, this needs a mention in the Contrib Guide.